File: 1 - INICIANTE/python/1074.py
n = int(input())
for c in range(0, n):
    m = int(input())
    if m == 0:
        print('NULL')
    elif (m > 0) and (m % 2 == 0):
        print('EVEN POSITIVE')
    elif (m > 0) and (m % 2 != 0):
        print('ODD POSITIVE')
    elif (m < 0) and (m % 2 == 0):
        print('EVEN NEGATIVE')
    else:
        print('ODD NEGATIVE')

File: 1 - INICIANTE/C++17/1074.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main() {
  
  int x,a; 
  int contador = 0;

  cin >> x;
 
  for (int i=0; i<x; i++) {
    cin >> a;
    
    if (a==0) {
    cout << "NULL" << "\n";
    }
    
    else if (a>0 && a%2==0) {
    cout << "EVEN POSITIVE" << "\n";
    }
    else if (a>0 && a%2!=0) {
    cout << "ODD POSITIVE" << "\n";
    } 
    else if (a<0 && a%2!=0) {
    cout << "ODD NEGATIVE" << "\n";
    }
    else if (a<0 && a%2==0) {
    cout << "EVEN NEGATIVE" << "\n";
    } 
    
        
  }

  return 0;
  }
